site/content/blog/2023-09-09-browser-extension/index.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Parsing .DS_Store files with ZAP"
+summary: "ZAP Spider can now probe and parse macOS' .DS_Store files."
+images:
+- https://www.zaproxy.org/blog/2023-09-09-browser-extension/images/zapgsoc.png
+type: post
+tags:
+- blog
+- browser extension
+- gsoc 2023
+date: "2023-09-09"
+authors:
+    - Aryan Gupta
+---
+# Browser Recorder - Exploring My Google Summer of Code 2023 Journey
+
+![ZAP GSoC Image](images/zapgsoc.png)
+
+## My Introduction
+
+I'm Aryan Gupta, a passionate software developer with a knack for problem-solving. I've worked with various programming languages, including Java, JavaScript, TypeScript, C++, and Python. As a seasoned full-stack developer well-versed in the MERN stack, I've always been eager to learn new things. My curiosity also extends to cybersecurity, where I'm fascinated by the challenge of uncovering online vulnerabilities.
+
+Contact Me Here: [GitHub](https://www,github.com/aryangupta701) | [LinkedIn](https://www.linkedin.com/in/aryan-gupta-78273a1b6/) | [Twitter](https://twitter.com/aryangupta_701)
```

Comment:
This info would be better on a Team page.
You can add a page for yourself here: https://www.zaproxy.org/docs/team/ (after Core Team, before Extended). You can do that in another PR which would get merged first if you like 😁

Reply:
okay